soc/nxp/imxrt/imxrt118x/soc.c

```diff
@@ -233,6 +233,20 @@ static ALWAYS_INLINE void clock_init(void)
 	CLOCK_SetRootClock(kCLOCK_Root_Lpspi0102, &rootCfg);
 #endif
 
+#if defined(CONFIG_COUNTER_MCUX_GPT) && \
```

Comment:
This should be split, into two `#if` statements, right? IE only clock GPT1 if the GPT1 nodelabel is enabled, and the same for GPT2

Reply:
Updated to split.

Reply:
Just a personal thing but I've seen this type of code structure do the following:

#if defined(CONFIG_COUNTER_MCUX_GPT)
#if (DT_NODE_HAS_STATUS(DT_NODELABEL(gpt1), okay))
/* insert code */
#endif /* DT_NODELABEL(gpt1) */

#if (DT_NODE_HAS_STATUS(DT_NODELABEL(gpt2), okay))
/* insert code */
#endif /* DT_NODELABEL(gpt2) */

#endif /* CONFIG_COUNTER_MCUX_GPT */

Reply:
Yes, it seems better for users, I will change them like comment above. Thank you.